boards/arm/stm32/stm3220g-eval/src/stm32_selectlcd.c

```diff
@@ -136,9 +136,9 @@ void stm32_selectlcd(void)
 
   /* Bank3 NOR/SRAM timing register configuration */
 
-  putreg32(FSMC_BTR_ADDSET(5) | FSMC_BTR_ADDHLD(0) |
-           FSMC_BTR_DATAST(9) | FSMC_BTR_BUSTURN(0) |
-           FSMC_BTR_CLKDIV(0) | FSMC_BTR_DATLAT(0) |
+  putreg32(FSMC_BTR_ADDSET(5) | FSMC_BTR_ADDHLD(1) |
+           FSMC_BTR_DATAST(9) | FSMC_BTR_BUSTURN(1) |
+           FSMC_BTR_CLKDIV(1) | FSMC_BTR_DATLAT(2) |
```

Comment:
Could you please write more about changes in timing configuration?

Reply:
The argument is subtracted by 1 or 2 in:
https://github.com/apache/incubator-nuttx/blob/master/arch/arm/src/stm32/hardware/stm32_fsmc.h#L160-L180
My change select the smallest value to avoid the result become to negative(huge positive which is definitely wrong) number.